<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestSetUp</class>
 <widget class="QWidget" name="TestSetUp">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1401</width>
    <height>792</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1401</width>
    <height>792</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1401</width>
    <height>792</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Verdana</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Versuchsaufbau</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <selectedon>icon materials/4.png</selectedon>
   </iconset>
  </property>
  <widget class="QLabel" name="Warning">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>720</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>51</width>
     <height>51</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>51</width>
     <height>51</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTabWidget" name="TestSetUpGroup">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>1361</width>
     <height>631</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1361</width>
     <height>631</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1361</width>
     <height>631</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Parameters Setup</string>
    </attribute>
    <widget class="QLabel" name="label_Polar">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
      </font>
     </property>
     <property name="text">
      <string>Polarization:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>55</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>Horizental</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Vertikal</string>
      </property>
     </item>
    </widget>
    <widget class="QCheckBox" name="checkBox_linearity">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>50</y>
       <width>211</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Verstärker Linearität checken</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_FieldToler">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>50</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Feld Toleranz:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_toler1">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>55</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>+4V/m</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_durch">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>55</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>/</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_toler2">
     <property name="geometry">
      <rect>
       <x>815</x>
       <y>55</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0V/m</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_new">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Neu</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_copy">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>150</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Kopieren</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_edit">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>150</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Bearbeiten</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_delete">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>150</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Löschen</string>
     </property>
    </widget>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>1310</width>
       <height>450</height>
      </rect>
     </property>
     <attribute name="headerVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="headerCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="headerDefaultSectionSize">
      <number>130</number>
     </attribute>
     <attribute name="headerHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="headerShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="headerStretchLastSection">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Freq (Hz)</string>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Freq Step</string>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Level</string>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Level Step</string>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Dwell</string>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>CW</string>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Modulation 1</string>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Modulation 2</string>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <item>
      <property name="text">
       <string>80M</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="text">
       <string>1%</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="text">
       <string>30</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="text">
       <string>None</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="text">
       <string>1s</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="text">
       <string>Off</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="text">
       <string>AM Sine(1kHz, 80%)</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="text">
       <string>Off</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1G</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="text">
       <string>1%</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="text">
       <string>30</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="text">
       <string>None</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="text">
       <string>1s</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="text">
       <string>Off</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="text">
       <string>Off</string>
      </property>
      <property name="text">
       <string>Off</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="TestSetUpSignalRouting" native="true">
    <attribute name="title">
     <string>Signal Routing</string>
    </attribute>
    <widget class="QTableWidget" name="SignalTabel">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>160</y>
       <width>1311</width>
       <height>451</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>1311</width>
       <height>451</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1311</width>
       <height>451</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <column>
      <property name="text">
       <string>Frequenz</string>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Gerätenamen</string>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Schalter</string>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Hilfs</string>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Benutzermeldung</string>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="NewButton">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>100</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>131</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>131</width>
       <height>41</height>
      </size>
     </property>
     <property name="text">
      <string>Neu</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CopyButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>100</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>131</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>131</width>
       <height>41</height>
      </size>
     </property>
     <property name="text">
      <string>Kopieren</string>
     </property>
    </widget>
    <widget class="QPushButton" name="EditButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>100</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>131</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>131</width>
       <height>41</height>
      </size>
     </property>
     <property name="text">
      <string>Bearbeiten</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DeleteButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>100</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>131</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>131</width>
       <height>41</height>
      </size>
     </property>
     <property name="text">
      <string>Löschen</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ImportButton">
     <property name="geometry">
      <rect>
       <x>1030</x>
       <y>100</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>131</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>131</width>
       <height>41</height>
      </size>
     </property>
     <property name="text">
      <string>Importieren</string>
     </property>
    </widget>
    <widget class="QPushButton" name="systemTestButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>1180</x>
       <y>100</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>151</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>151</width>
       <height>41</height>
      </size>
     </property>
     <property name="text">
      <string>System Testen</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="ExpandButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>171</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>171</width>
       <height>41</height>
      </size>
     </property>
     <property name="font">
      <font>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>+ Aller erweitern</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CollposeButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>100</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>241</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>241</width>
       <height>41</height>
      </size>
     </property>
     <property name="font">
      <font>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>- Alle zusammenbrechen</string>
     </property>
    </widget>
    <widget class="QLabel" name="SystemControllerLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>250</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>31</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Aktive Systemkontroller:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="KontrollerComBoBox">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>50</y>
       <width>1011</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>1011</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1011</width>
       <height>31</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Wählen Sie alle Systemkontroller aus, die diesem Testaufbau zugeordnet sind.</string>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="currentText">
      <string>None</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>0</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Signal Routing</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="WarningLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>730</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>251</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>251</width>
     <height>31</height>
    </size>
   </property>
   <property name="text">
    <string>Das Signal Routing ist leer!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ApplyButton">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>720</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>131</width>
     <height>41</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>131</width>
     <height>41</height>
    </size>
   </property>
   <property name="text">
    <string>Anwenden</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>720</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>131</width>
     <height>41</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>131</width>
     <height>41</height>
    </size>
   </property>
   <property name="text">
    <string>Stornieren</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>CancelButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>TestSetUp</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1274</x>
     <y>748</y>
    </hint>
    <hint type="destinationlabel">
     <x>973</x>
     <y>736</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
